plugins/env/test/samson_env/samson_plugin_test.rb

```diff
@@ -4,15 +4,20 @@
 SingleCov.covered! uncovered: 3
 
 describe SamsonEnv do
-  let(:deploy) { deploys(:succeeded_test) }
-  let(:stage) { deploy.stage }
-  let(:project) { stage.project }
+  let(:deploy) {deploys(:succeeded_test)}
```

Comment:
Same rubocop issue here, might be issue with your settings in RubyMine

Reply:
done rubocop passes